snykTask/src/task-lib.ts

```diff
@@ -3,6 +3,7 @@ import * as tr from 'azure-pipelines-task-lib/toolrunner';
 import * as tl from 'azure-pipelines-task-lib/task';
 import stream = require('stream');
 import * as fs from 'fs';
+const fsPromises = require('fs').promises;
```

Comment:
Let's stick with import syntax, as it's in other imports:
`import * as fsPromises from 'fs/promises'`.